packages/react-native/ReactCommon/react/bridgeless/platform/ios/Core/RCTInstance.mm

```diff
@@ -37,6 +36,7 @@
 #import <react/utils/ContextContainer.h>
 #import <react/utils/ManagedObjectWrapper.h>
 
+#import "../NativeViewConfig/RCTLegacyUIManagerConstantsProvider.h"
```

Comment:
We should avoid relative imports. When the headers are moved around, this path can break.